docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+# Pull Request Template
+
+## Description
+
+:question: Please include a summary of the change(s), a small paragraph in your own words is fine.
+
+
+:question: Breakdown of Task, steps taken.
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+:question: To complete these changes, have you used any tutorials or done any reading which would be of benefit to the group? If yes, you can add the links here. It is great for our growth, but it can also aid in troubleshooting.
```

Comment:
@EbonyChantelle I re-worded it, check it out to see if it sounds better now.

Reply:
Change to the first and last sentences slightly to the following and then we are good to go....

To complete the changes, did you use any tutorials or do any reading that would be of benefit to the group?

It is great for our growth, and can also aid with troubleshooting.

Reply:
okay